src/utils/formHelpers.ts

```diff
@@ -45,6 +81,39 @@ class FormHelpers {
 		return messages.join(" ");
 	};
 
+	checkPasswordStrength = (password: string, messages: string[]) => {
+		if (passwordStrength(password, defaultOptions).value === "Too weak") {
+			messages.push(errorPasswordStrengthTooWeak);
+		}
+		if (passwordStrength(password, defaultOptions).value === "Weak") {
+			messages.push(errorPasswordStrengthWeak);
+		}
+		if (passwordStrength(password, defaultOptions).value === "Medium") {
+			messages.push(errorPasswordStrengthMedium);
+		}
+		return messages.join(" ");
+	};
+
+	checkPasswordSameSymbolsThreeNumbersPrivateBirth = (password: string, messages: string[]) => {
+		const passArray = password.split("");
+		for (let i = 0; i < password.length; i++) {
+			if (passArray[i] === passArray[i + 1]) {
+				messages.push(errorPasswordMoreTwoSameSymbolsRepeat);
+			}
+			if (+passArray[i + 1] - +passArray[i] == 1 && +passArray[i + 2] - +passArray[i + 1] == 1) {
+				messages.push(errorPasswordThreeNumbersRow);
+			}
+			if (
+				password.match(/\d{4}(-|\/|\.)\d{2}(-|\/|\.)\d{2}/g) ||
+				password.match(/\d{2}(-|\/|\.)\d{2}(-|\/|\.)\d{4}/g) ||
+				password.match(/\d{8}/g)
+			) {
+				return messages.push(errorPasswordPrivateBirthDate);
+			}
```

Comment:
better replace processing of these moments with static check using regular expressions (parts of regular expressions) and move it to constructPasswordMessage. For example,

if (password.length >= minPasswordLength && (!/(?=.*\d)/).test(password)) {
messages.push(errorPasswordNumber);